ui/src/builder/BuilderCopyText.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+	<span class="tooltip">
```

Comment:
If we can center it on long ids that'd be good, but not a deal breaker at all.

<img width="227" alt="Screenshot 2024-03-12 at 16 11 23" src="https://github.com/streamsync-cloud/streamsync/assets/64783088/5a33e5b5-6b92-4978-b90d-ab6d97e4c3a7">

Reply:
It is centered to span containing this text, only thing I can do here is adding:
```
white-space: nowrap;
```
but not sure if this is the way?